pkg/quota/quota.go

```diff
@@ -14,7 +15,8 @@ const (
 var (
 	// UseQuota represents use quota or not.
 	UseQuota = true
-	gquota   = NewQuota("")
+	// Gquota represents global quota.
+	Gquota = NewQuota("")
```

Comment:
At the first glance, Gquota likes grpquota, is there another name?

Reply:
it means global quota object.:)